index.js

```diff
@@ -115,6 +118,32 @@ cm.Item({
   }
 });
 
+cm.Item({
+  label: 'Send to mini player',
+  context: cm.SelectorContext('[href*="vine.co/v/"]'),
+  contentScript: 'self.on("click", function (node, data) {' +
+                 '  self.postMessage(node.href);' +
+                 '});',
+  onMessage: function(url) {
+    const id = getVineId(url);
+    updatePanel({domain: 'vine.co', id: id, src: ''});
+    getVineUrl(url, function(err, streamUrl) {
+      if (!err) updatePanel({src: streamUrl});
+    });
+  }
+});
+
+function getVineId(str) {
```

Comment:
When I do `$ npm i get-video-id@latest`, I get the correct version with code in the correct places. Mysteries... ?

For this, I removed code from index.js, updated the packages.json, packaged a new .xpi, and let the maintainer of the module know about the version thing.